articles/osquery-evented-tables-overview.md

```diff
@@ -0,0 +1,161 @@
+
+To use the `es_process_events` tables, use the flag `--disable_endpointsecurity=false` see [these instructions](https://osquery.readthedocs.io/en/latest/deployment/process-auditing/#auditing-processes-with-endpointsecurity). To use `process_events` and `socket_events` with OpenBSM, see [these instructions](https://osquery.readthedocs.io/en/latest/deployment/process-auditing/#auditing-processes-with-openbsm). 
+
+#### Windows
+Currently, osquery does not support process auditing for Windows. To learn more about process auditing on Windows, visit [here](https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/security-auditing-overview). Fleet is tracking work to build process auditing for Windows in osquery [here](https://github.com/fleetdm/fleet/issues/7732).
+
+### YARA scanning
+[YARA](https://virustotal.github.io/yara/) is a malware research and detection tool available on linux and MacOS that allows users to create descriptions of malware families based on patterns of text or binary code. Each potential piece of malware is matched against a YARA rule and triggers if the specified conditions are met. 
+
+Osquery applies pre-specified YARA rules to incoming events in the `file_events` table to populate the `yara_events` table. As such, it requires the following flags:
+* `--disable_events=false`
+* `--enable_file_events=true`
+
+With the appropriate flags set, then specify the appropriate YARA rules in the osquery configuration using the format as described [here](https://osquery.readthedocs.io/en/stable/deployment/yara/#yara-configuration).
+
+More topics and useful links for YARA:
+* [Osquery's YARA scanning guide](https://osquery.readthedocs.io/en/stable/deployment/yara/#continuous-monitoring-using-the-yara_events-table)
+* [How to write YARA rules](https://yara.readthedocs.io/en/stable/writingrules.html)
+* [Repository of example YARA rules](https://github.com/Yara-Rules/rules)
+* [Collection of use YARA resources by InQuest](https://github.com/InQuest/awesome-yara)
+* [YARA performance guidelines](https://github.com/Neo23x0/YARA-Performance-Guidelines/)
+
+### Other tables
+These other event tables are also available in osquery. We will provide more information for them in other guides. 
+
+| Table name | OS | Flags |
+| :- | :-- | :-- |
+| apparmor_events | linux | --audit_allow_apparmor_events=true |
+| disk_events | MacOS
+| hardware_events | MacOS, linux | 
+| seccomp_events | linux |
+| selinux_events | linux | --audit_allow_selinux_events=true |
+| syslog_events | linux | 
+| user_interaction_events | MacOS |
+| user_events | Linux | --audit_allow_user_events=true |
+| ??? | ??? | --audit_allow_fork_process_events=true |
+| ??? | ??? | --audit_allow_kill_process_events=true |
```

Comment:
This is both for `process_events` table on Linux. `--audit_allow_fork_process_events` will enable capturing `fork` syscall, and `--audit_allow_kill_process_events` will enable capturing `kill` syscall